dask/array/tests/test_fft.py

```diff
@@ -131,25 +131,24 @@ def test_wrap_ffts(modname, funcname, dtype):
     except AttributeError:
         pytest.skip("`%s` missing function `%s`." % (modname, funcname))
 
-    if modname == "numpy.fft" and funcname == "irfft":
-        if dtype == "float32":
-            dtype = "complex64"
-        elif dtype == "float64":
-            dtype = "complex128"
-
     darrc = darr.astype(dtype)
     darr2c = darr2.astype(dtype)
     nparrc = nparr.astype(dtype)
 
-    wfunc = fft_wrap(func)
-    assert wfunc(darrc).dtype == func(nparrc).dtype
-    assert wfunc(darrc).shape == func(nparrc).shape
-    assert_eq(wfunc(darrc), func(nparrc))
-    assert_eq(wfunc(darrc, axis=1), func(nparrc, axis=1))
-    assert_eq(wfunc(darr2c, axis=0), func(nparrc, axis=0))
-    assert_eq(wfunc(darrc, n=len(darrc) - 1),
-              func(nparrc, n=len(darrc) - 1))
-    assert_eq(wfunc(darrc, axis=1, n=darrc.shape[1] - 1),
-              func(nparrc, n=darrc.shape[1] - 1))
-    assert_eq(wfunc(darr2c, axis=0, n=darr2c.shape[0] - 1),
-              func(nparrc, axis=0, n=darr2c.shape[0] - 1))
+    if modname == "scipy.fftpack":
+        if "rfft" in funcname:
+            with pytest.raises(ValueError):
+                wfunc = fft_wrap(func)
```

Comment:
These two if statements should be merged into one line.  Currently you don't test any of the scipy functions except rfft